x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -121,20 +121,7 @@ export const ENTITY_ANALYTICS_PATH = '/entity_analytics' as const;
 export const ENTITY_ANALYTICS_MANAGEMENT_PATH = `/entity_analytics_management` as const;
 export const ENTITY_ANALYTICS_ASSET_CRITICALITY_PATH =
   `/entity_analytics_asset_criticality` as const;
-export const APP_OVERVIEW_PATH = `${APP_PATH}${OVERVIEW_PATH}` as const;
```

Comment:
I'm assuming all of the deleted ones are not in use anymore?

Reply:
yup